src/parser/content-evaluator-module.ts

```diff
@@ -127,7 +127,7 @@ export class ContentEvaluatorModule implements Module {
       currentComment.score = {
         ...(currentComment.score || { multiplier: 0 }),
         relevance: new Decimal(currentRelevance).toNumber(),
-        reward: currentReward.toNumber(),
+        reward: new Decimal(currentReward).mul(100).round().div(100).toNumber(),
```

Comment:
Multiplying and dividing seems wrong. It just cancels itself out.

Reply:
@0x4007 It doesn't because of the round, so this basically is a 3 decimal rounding operation. But I don't think this is necessary because Decimaljs can handle an "infinite" amount of decimals. `currentReward` should be a finite number during all operations.

Reply:
Just to clarify, are you suggesting that rounding should be avoided altogether to keep maximum precision until the final display stage? I understood from your comment that decimal.js can handle high precision throughout all operations without the need for intermediate rounding. If so, would it be preferable to apply formatting only at the end for display purposes, rather than during calculations?

Reply:
Yes exactly.